lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -477,6 +477,50 @@ void testMain() {
 
   // ALL ADAPTERS
 
+  _testEach(
```

Comment:
Let's leave a comment explaining why we test this. IIRC, we have two use-cases:

- The native text field.
- Platform views: there are cases when a platform view wants to handle an event and not propagate it to Flutter, but because Flutter listens during the capture phase, by the time the event reaches the platform view it's too late to stop propagation.

Reply:
Done